addon/templates/components/uni-country.hbs

```diff
@@ -0,0 +1,12 @@
+{{#uni-autocomplete
+  options=countries
+  value=selectedCountry
+  placeholderText=placeholderText
+  hasError=hasError
+  noResultsComponent="uni-autocomplete-no-results"
+  noResultsText=noResultsText
+  searchTextValues=(action "countryNames")
+  onSelected=(action "changeCountryCode") as |optionSearchable|}}
+  {{inline-svg (i "flags/${code}" code=optionSearchable.option.code)}}
+  {{titleize optionSearchable.matchedValues.firstObject}}
```

Comment:
Add an if to allow the block option, where it yields the option

Reply:
why?

Reply:
So when we want to have so other component or html as an option, it's easier to do that :) It makes the component more flexible